f/nametag/include/f_nametagBrief.sqf

```diff
@@ -14,7 +14,7 @@
 	waitUntil {scriptDone f_script_briefing};
 
 	_bstr = format ["<font size='18'>WH NAMETAGS</font><br/><br/>Toggle name tags for friendly units by pressing %1.<br/><br/>Name tags are displayed when aiming at individual units up to and above %4m away, and constantly for all units within %3m.<br/><br/>Note that using CBA will disable this menu and replace it with an enchanced one under ADDON OPTIONS.
```

Comment:
Shouldn't this line be updated as well to account for the different number of arguments?
e.g. change %3 and %4 to %2 and %3 respectively